pkgs/tools/misc/tlp/default.nix

```diff
@@ -87,8 +87,8 @@
         smartmontools
         systemd
         utillinux
-        x86_energy_perf_policy
       ] ++ lib.optional enableRDW networkmanager
+        ++ lib.optional (lib.any (lib.meta.platformMatch stdenv.hostPlatform) x86_energy_perf_policy.meta.platforms) x86_energy_perf_policy
```

Comment:
Why do you need `lib.any` here? Why not just `x86_energy_perf_policy.meta.platforms`?

Reply:
My intention was to say that if the current platform matches any of the platforms supported by `x86_energy_perf_policy`, then include it, otherwise don't. So I guess `lib.any` is there because there are multiple platforms in `x86_energy_perf_policy.meta.platforms`.

Reply:
Understood, approved!